### PR TITLE
add openapi-spec-validator

### PR DESCRIPTION
**Pull request by rvalieris**
this package is required to update [connexion](https://github.com/conda-forge/connexion-feedstock) to the latest version.

Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/openapi-spec-validator/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  version: {{ version }}
+
+source:
+  fn: {{ version }}.tar.gz
+  url: https://github.com/p1c2u/openapi-spec-validator/archive/{{ version }}.tar.gz
+  sha256: 49a0c4dbd89fa66737fd8e93d71071d93c65c8586a4bff89771c65ec897916af
+
+build:
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
```
Comment (scopatz) [inline recipes/openapi-spec-validator/meta.yaml]: This seems like it can be `noarch: python`

Comment (rvalieris) [reply recipes/openapi-spec-validator/meta.yaml]: I did tried with noarch above but the windows build failed with:

`[noarch_python] Noarch package contains binary script: openapi-spec-validator.exe`

Comment (beckermr) [reply recipes/openapi-spec-validator/meta.yaml]: I'm ok without noarch here. Will merge in a day or so if we don't hear back from someone else on @conda-forge/staged-recipes.

Comment (chrisburr) [reply recipes/openapi-spec-validator/meta.yaml]: That error means you’re missing console script entry points:
https://github.com/p1c2u/openapi-spec-validator/blob/master/setup.py#L69
https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#python-entry-points

Comment (rvalieris) [reply recipes/openapi-spec-validator/meta.yaml]: looks like it worked ! thanks @chrisburr .

**Pull request merged**